sigpyproc/Filterbank.py

```diff
@@ -102,10 +91,10 @@ def invertFreq(self, gulp=512, start=0, nsamps=None, filename=None,
                                           nbits=self.header.nbits,
                                           back_compatible=back_compatible)
         for nsamps, ii, data in self.readPlan(gulp, start=start, nsamps=nsamps, **kwargs):
-            self.lib.invertFreq(as_c(data),
-                                as_c(out_ar), 
-                                C.c_int(self.header.nchans),
-                                C.c_int(nsamps))
+            lib.invertFreq(data,
```

Comment:
wondering whether you can push these back onto one line now that the C wrappers are no more

Reply:
(likewise with other methods, probably more 'PEP8' compliant)

Reply:
Yes, will properly format after merging this PR.